doc/api/sqlite.md

```diff
@@ -88,7 +88,7 @@ exposed by this class execute synchronously.
 added: v22.5.0
```

Comment:
I think we should add changelog entries to these functions mentioning this change.

Reply:
Good point. Will do.